Doc/library/fileinput.rst

```diff
@@ -171,8 +170,8 @@ available for subclassing as well:
    .. versionchanged:: 3.2
       Can be used as a context manager.
 
-   .. deprecated:: 3.8
-      Support for :meth:`__getitem__` method is deprecated.
+   .. deprecated-removed:: 3.8 3.11
```

Comment:
```suggestion
   .. versionchanged:: 3.11
```
and move it below `versionchanged:: 3.8`.

`deprecated-removed` is only used for future planned removing. After removing it is just `versionchanged`.